apps/extension-wallet/src/stores/settings.ts

```
@@ -118,6 +131,24 @@ export const useSettingsStore = create<SettingsState>()(
             typeof persisted.requirePasswordForSensitiveActions === 'boolean'
               ? persisted.requirePasswordForSensitiveActions
               : DEFAULTS.requirePasswordForSensitiveActions,
+          dailyTransferLimit:
+            typeof dailyTransferLimit === 'number' && dailyTransferLimit >= 0
+              ? dailyTransferLimit
+              : DEFAULTS.dailyTransferLimit,
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)